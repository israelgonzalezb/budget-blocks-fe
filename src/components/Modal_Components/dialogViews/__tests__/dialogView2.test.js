
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import dialogView2 from '../dialogView2';


describe('dialogView2 test', () => {
  it('dialogView2 should match snapshot', () => {
    const component = renderer.create(<dialogView2
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // hey there...
  });
});
