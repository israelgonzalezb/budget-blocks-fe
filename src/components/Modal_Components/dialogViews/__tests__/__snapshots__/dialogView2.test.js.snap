// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dialogView2 test dialogView2 should match snapshot 1`] = `<dialogView2 />`;
